SchoolHousing/SchoolHousing/Pages/Index.cshtml.cs

```diff
@@ -10,17 +10,25 @@
 using ElementarySchoolResponse;
 public class IndexModel : PageModel
 {
-    public void OnGet()
+    public JsonResult OnGet()
```

Comment:
This is changing functionality.  As the group has it now, an HTML page is returned.  You're changing this to return JSON.  The group may prefer what they had originally.